packages/dbos-cloud/cli.ts

```diff
@@ -139,16 +139,17 @@ applicationCommands
   .command('deploy')
   .description('Deploy this application to the cloud and run associated database migration commands')
   .option('--verbose', 'Verbose log of deployment step')
-  .action(async (options: {verbose?: boolean}) => {
-    const exitCode = await deployAppCode(DBOSCloudHost, false, options.verbose ?? false);
+  .option('-p, --previous-version <string>', 'Specify a previous version to restore')
```

Comment:
Can this just be `-V, --version`? Because I assume you can restore to a previous version, and then restore back to a later version.

Reply:
`-V` flag is overloaded. Prefer `previous-version` to make it clear this is about restoring a previous version.